context/app/static/js/components/Detail/visualization/VisualizationShareButton/constants.js

```diff
@@ -0,0 +1,5 @@
+const DEFAULT_LONG_URL_WARNING = 'Warning: Long URLs may not work on some browsers.';
+
+const DEFAULT_EMAIL_MESSAGE = 'Here is an interesting dataset I found in the HuBMAP Data Portal:';
+
+export { DEFAULT_LONG_URL_WARNING, DEFAULT_EMAIL_MESSAGE };
```

Comment:
Not sure we gain that much by splitting this out.

Reply:
I'm in general a fan of "constants" files especially because the number is now going to go up again if we show a message for FF.